### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by legokichi**
- bump rust_toolchain version
- clippy
- udate outdated crates

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -7,39 +7,39 @@ use std::net::IpAddr;
 use std::path::PathBuf;
 
 use log::*;
-use structopt::*;
 
 use gatekeeper as gk;
 
-#[derive(StructOpt, Debug)]
-#[structopt(name = "gatekeeper")]
+#[derive(clap::Parser, Debug)]
```
Comment (eldesh) [inline src/main.rs:13]: `-V` option seems to be deleted.

```
gatekeeper$ cargo run -- -V
   Compiling gatekeeper v2.2.0 (/home/eldesh/work/gatekeeper)
    Finished dev [unoptimized + debuginfo] target(s) in 3.27s
     Running `target/debug/gatekeeperd -V`
gatekeeperd
error: unexpected argument '-V' found

Usage: gatekeeperd [OPTIONS]

For more information, try '--help'.
```

**Review by eldesh — CHANGES_REQUESTED** [review]
commented

File: src/main.rs (hunk 1 above)
Comment (legokichi) [reply src/main.rs:13]: https://github.com/Idein/gatekeeper/pull/61/commits/21d64f3c42b5f8349ef313408230e4e350551193
fixed

**Review by eldesh — APPROVED** [review]
Looks good :+1:

**Review by eldesh — CHANGES_REQUESTED** [review]
Some unit tests are failed.
https://app.circleci.com/pipelines/github/Idein/gatekeeper/196/workflows/3a6b07ad-69b5-406e-87ef-1472b0a6e8ff/jobs/434

**Review by eldesh — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Review by garasubo — APPROVED** [review]

**Review by eldesh — APPROVED** [review]
:ok:

**Pull request merged**